components/payments/cmd/connectors/internal/api/connector.go

```diff
@@ -124,6 +125,38 @@ func listTasks[Config models.ConnectorConfigObject](
 	}
 }
 
+func webhooks[Config models.ConnectorConfigObject](
+	b backend.ManagerBackend[Config],
+	apiVersion APIVersion,
+) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		connectorID, err := getConnectorID(b, r, apiVersion)
+		if err != nil {
+			api.BadRequest(w, ErrInvalidID, err)
+			return
+		}
+
+		body, err := io.ReadAll(r.Body)
+		if err != nil {
+			api.BadRequest(w, ErrMissingOrInvalidBody, err)
+		}
+		defer r.Body.Close()
+
+		err = b.GetManager().HandleWebhook(r.Context(), &models.Webhook{
+			ID:          uuid.New(),
+			ConnectorID: connectorID,
+			RequestBody: body,
+		})
+		if err != nil {
+			handleConnectorsManagerErrors(w, r, err)
+			return
+		}
+
+		w.WriteHeader(http.StatusOK)
```

Comment:
Maybe you should process the webhook asynchornously and respond with a 200 as soon as possible.
webhooks are often time limited (just a suggestion).

Reply:
It's already the case since the task is launched async !